x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/complete/complete.spec.ts

```diff
@@ -261,12 +250,12 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
       let events: StreamingChatResponseEvent[];
 
       before(async () => {
-        events = await getEvents({}, 'Title for at new conversation', 'Hello again').then(
-          (_events) => {
-            return _events.filter(
-              (event) => event.type !== StreamingChatResponseEventType.BufferFlush
-            );
-          }
+        void proxy.interceptTitle('Title for at new conversation');
```

Comment:
Not introduced by this PR, but maybe we can update it.

```suggestion
        void proxy.interceptTitle('Title for a new conversation');
```